cmd/ocm/gcp/update-wif-config.go

```diff
@@ -103,6 +110,25 @@ func updateWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) e
 		wifBuilder.WifTemplates(append(existingTemplates, wifTemplate)...)
 	}
 
+	// Create the client for the GCP API
+	gcpClient, err := gcp.NewGcpClient(ctx)
```

Comment:
gcpClient creation should also be moved into the if clause , i think there is no need for gcpClient if federate project is null